addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -255,6 +285,15 @@ function factory(dependencies) {
         messages: many2many('mail.message', {
             related: 'threadCache.messages',
         }),
+        messaging: many2one('mail.messaging', {
+            compute: '_computeMessaging',
+        }),
+        /**
+         * Serves as compute dependency.
+         */
+        messagingDeviceIsMobile: attr({
```

Comment:
If you have a field called just `device` this can be just `deviceIsMobile` (drop messaging from the name, and check alphabetrical order after).